packages/sortable-behavior-bundle/README.md

```diff
@@ -32,6 +32,31 @@ return [
 ];
 ```
 
+### Use a draggable list instead of up/down buttons
+
+In order to use a draggable list instead of up/down buttons, change the template in the `move` action to `@RunroomSortableBehavior/sort.html.twig`.
+
+```php
+<?php
+
+    // ClientAdmin.php
+    protected function configureListFields(ListMapper $listMapper)
+    {
+        $listMapper
+            ->add('_action', null, array(
+                'actions' => array(
+                    'move' => array(
+                        'template' => '@RunroomSortableBehavior/sort.html.twig',
+                        'enable_top_bottom_buttons' => true, //optional
+                    ),
+                ),
+            ))
+        ;
+    }
+```
+### Submitting a Pull Request or an Issue
```

Comment:
This change should be made on each Readme (for all the subpackages) and imo it should be placed at an earlier point on the readme. 

Also I cant really disable PRs for subpackages, or at least I do not see the option on the github interface, do you know how to do it?

Reply:
>Also I cant really disable PRs for subpackages, or at least I do not see the option on the github interface, do you know how to do it?
Didn't find it, so we left it as is
> it should be placed at an earlier point on the readme.
Before what section to place it? I can suggest the second position (right after installation)

Reply:
I have taken a look at other mono repo (built with the same tool than this): https://github.com/symplify/monorepo-builder

I think we could place the same line before License section. can you do it for all readmes? (Except the root one)

Reply:
Done, please check